src/popup/router/components/Modals/AccountDetails.vue

```diff
@@ -11,7 +11,7 @@
         <AccountInfo
           v-bind="$attrs"
```

Comment:
I suggest to not pass all attrs to child components as this creates undiscoverable layer of abstraction. We should be aware what data we are passing.